sites/main-site/src/content/events/2025/hackathon-march-2025/university-college-london.mdx

```diff
@@ -19,9 +19,34 @@ layout: '@layouts/events/HackathonMarch2025.astro'
 
 Attendees from outside University College London **can** attend.
 
-Main contact: Chris Wyatt ([ucbtcdr@ucl.ac.uk](mailto:ucbtcdr@ucl.ac.uk))
+Main contact:
+Chris Wyatt ([ucbtcdr@ucl.ac.uk](mailto:ucbtcdr@ucl.ac.uk)),
+Fernando Duarte ([ucbtfrd@ucl.ac.uk](mailto:ucbtfrd@ucl.ac.uk))
 
-Venue not yet booked, please check back later for more information.
+## Site Specific guidelines
 
-Can currently only find a room for Monday 9-6pm and Tuesday 9-1pm.
-Trying to find alternatives for the other day/times.
+We are hosting a site at the new UCL EAST campus in Stratford.
+
+It has a capacity of 22 people. We hope to have lunch sponsored on one of the days, plus refreshments.
+
+Registration is required, and will open on the January 31st.
+
+Welcome to all!
+(with limits on numbers of people, first come, first served policy).
+
+More information to follow soon. Hope you can make it!
+It is a great place to meet with fellow London (and further) based Nextflow enthuiasts
+
+## Guidelines
+
+Please familiarize yourself with the [hackathon guidelines](https://nf-co.re/events/2025/hackathon-march-2025#in-person-local-sites)
+
+## Alternative overflow location (TBC)
+
+In case we get a huge response, we have an alternative additional option at the Bloomsbury campus (TBD).
+
+**24th:** 188 Tottenham Ct Rd (Room 4). Capacity = 40
+
+**25th:** LG04 Otto Koenigsberger & LG02 Patrick Abercrombie, Central House. Capacity = 20
+
+**26th:** 302 & 402 Tutorial Room, Bentham House. Capacity = 15
```

Comment:
```suggestion
In case we get a huge response, we have an alternative additional options at the Bloomsbury campus (TBC):

- **24th:** 188 Tottenham Ct Rd (Room 4). Capacity = 40
- **25th:** LG04 Otto Koenigsberger & LG02 Patrick Abercrombie, Central House. Capacity = 20
- **26th:** 302 & 402 Tutorial Room, Bentham House. Capacity = 15
```